browser/resources/federated_internals/federated_internals.js

```diff
@@ -9,9 +9,9 @@ function getProxy() {
 }
 
 function initialize() {
-  getProxy().getAdStoreInfo();
+  getProxy().updateDataStoresInfo();
```

Comment:
nit: Do we need `Info`

Reply:
We are updating the data store information in the internals page, we're not technically updating the data stores. But other than that don't have a strong preference